manifest/armv7l/a/a2ps.filelist

```diff
@@ -71,32 +72,32 @@
 /usr/local/share/a2ps/encoding/ms-cp1250.edf
 /usr/local/share/a2ps/encoding/ms-cp1251.edf
 /usr/local/share/a2ps/fonts/pcfont.pfa
+/usr/local/share/a2ps/ppd/README
 /usr/local/share/a2ps/ppd/level1.ppd
```

Comment:
Shouldn't README come after level1.ppd?

Reply:
I guess it being in all caps means it is sorted first?

Reply:
I suppose the ascii value is lower so that would make sense.

Reply:
We could use `LC_ALL=C sort -f` if we want to ignore case when sorting, I think.